test/functional/test_framework/test_framework.py

```
@@ -624,27 +624,13 @@ def dynamically_initialize_datadir(self, node_p2p_port, node_rpc_port):
-            f.write("listenonion=0\n")
-            f.write("printtoconsole=0\n")
-            f.write("upnp=0\n")
-            f.write("natpmp=0\n")
-            f.write("shrinkdebugfile=0\n")
-            f.write("dip3params=2:2\n")
-            f.write(f"testactivationheight=v20@{self.v20_height}\n")
-            f.write(f"testactivationheight=mn_rr@{self.mn_rr_height}\n")
-            os.makedirs(os.path.join(new_data_dir, 'stderr'), exist_ok=True)
-            os.makedirs(os.path.join(new_data_dir, 'stdout'), exist_ok=True)
+        write_config(os.path.join(new_data_dir, "dash.conf"),
+                n=mnidx, chain=self.chain, disable_autoconnect=self.disable_autoconnect)
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)